keps/sig-apps/2879-ready-pods-job-status/README.md

```diff
@@ -259,7 +262,7 @@ the controller doesn't create new Pods or tracks finishing Pods.
 ###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
 
 - [x] Metrics
-  - Metric name: `job_sync_duration_seconds`, `job_sync_total`.
+  - Metric name: `job_sync_duration_seconds`, `job_syncs_total`.
```

Comment:
Adjusted the metric name to the actual value: https://github.com/kubernetes/kubernetes/blob/e8abe1af8dcb36f65ef7aa7135d4664b3db90e89/pkg/controller/job/metrics/metrics.go#L52